src/Hangfire.Mongo/Dto/JobDto.cs

```diff
@@ -8,8 +9,7 @@ namespace Hangfire.Mongo.Dto
 #pragma warning disable 1591
     public class JobDto
     {
-        [BsonId(IdGenerator = typeof(StringObjectIdGenerator))]
-        public string Id { get; set; }
+        public ObjectId Id { get; set; }
```

Comment:
Here you have removed the BsonId attribute. Let's try to be consequent and do the same everywhere

Reply:
Ok